judo-ui-react/src/main/resources/actor/src/components/dialog/FilterDialog.tsx.hbs

```diff
@@ -118,7 +118,7 @@ const FilterOperator = ({ filter, operatorId, valueId, setFilterOperator }: Filt
       onChange={onChangeHandler}
     >
       {getOperatorsByFilter(filter).map((item) => (
-        <MenuItem id={valueId} key={item} value={item}>
+        <MenuItem id={valueId} key={item} value={item} className="filter-operation-item">
```

Comment:
Should be the className attribute after the React element just for the consistency